XMLtoMD/GuidelineXmlToMD/MarkdownOut/MdText.cs

```diff
@@ -0,0 +1,220 @@
+
+        /// <summary>
+        /// The Markdown heading 6 prefix string.
+        /// </summary>
+        public static readonly string Heading6Prefix = "###### ";
+
+        /// <summary>
+        /// The Markdown quote prefix string.
+        /// </summary>
+        public static readonly string QuotePrefix = "> ";
+
+        /// <summary>
+        /// The Markdown unordered list item prefix string.
+        /// </summary>
+        public static readonly string UnorderedListItemPrefix = "- ";
+
+
+        /// <summary>
+        /// The default number used for ordered list items.
+        /// </summary>
+        public static readonly int DefaultListItemNumber = 1;
+
+        /// <summary>
+        /// The Markdown ordered list item prefix string.
+        /// </summary>
+        public static readonly string OrderedListItemPrefix = DefaultListItemNumber + ". ";
+
+        /// <summary>
+        /// The Markdown indent string for sublists.
+        /// </summary>
+        public static readonly string ListItemIndent = new string(' ', 5);
+
+        /// <summary>
+        /// Styles the provided text using the specified <see cref="MdStyle"/>.
+        /// </summary>
+        /// <param name="text">The text to style.</param>
+        /// <param name="style">The Markdown style to apply.</param>
+        /// <returns>The styled string.</returns>
+        /// <exception cref="System.ArgumentException">The style is not recognized.</exception>
+        public static string Style(object text, MdStyle style) {
+            string wrap;
+            switch (style) {
+                case MdStyle.None: wrap = String.Empty; break;
+                case MdStyle.Italic: wrap = ItalicWrap; break;
+                case MdStyle.Bold: wrap = BoldWrap; break;
+                case MdStyle.BoldItalic: wrap = ItalicWrap + BoldWrap; break;
+                case MdStyle.Code: wrap = CodeWrap; break;
+                case MdStyle.StrikeThrough: wrap = StrikeThroughWrap; break;
+                default: throw new ArgumentException("The style is not recognized.");
+            }
+            return wrap + text + wrap;
+        }
+
+        /// <summary>
+        /// Formats the provided text using the specified <see cref="MdFormat"/>. If styling is also
+        /// desired, use the <see cref="StyleAndFormat"/> method or style the text first; formatting
+        /// text and then styling it produces output that may not parse correctly.
+        /// </summary>
+        /// <param name="text">The text to format.</param>
+        /// <param name="format">The Markdown format to apply.</param>
+        /// <returns>The formatted string.</returns>
+        /// <exception cref="System.ArgumentException">The format is not recognized.</exception>
+        public static string Format(object text, MdFormat format) {
+            string prefix;
+            switch (format) {
+                case MdFormat.None: prefix = String.Empty; break;
```

Comment:
nit: Use keyword over BCL type
```suggestion
                case MdFormat.None: prefix = string.Empty; break;
```